Cordova_QA_App/optimove.json

```diff
@@ -0,0 +1,4 @@
+{
+ 	 "optimove_credentials": "WzEsIm15LXRva2VuIiwibXktY29uZmlnLWZpbGUiXQ==",
+	"optimove_mobile_credentials": "WzEsImV1LWNlbnRyYWwtMSIsIjE5ZGY1ZTc3LTJhNmMtNGIzZC05YmI5LWNiNTQ0MDgwMmEyNiIsIlMxTksyWjBMSVpWeVl6eVk4bnB0RlAzQjBpSDVjbU50RU5pSSJd"
```

Comment:
We shouldnt ship credentials - possibly worth checking how QA apps for Android / IOS bake these in.

Reply:
the json file was changed  to {
 	 "optimove_credentials": "YOUR_OPTIMOVE_CREDENTIALS",
	"optimove_mobile_credentials": "YOUR_OPTIMOVE_MOBILE_CREDENTIALS"
}

Reply:
Do we need to consider scrubbing these from git history? Maybe trickier now it's on the PR diffs.